docs/sprints.md

```diff
@@ -38,26 +39,27 @@
 - Gerenciamento_turmas.html que conterá todas as turmas já criadas com ícone de edição e de exclusão em cada um deles.
 - Deverá ser criado um campo onde serão inseridas as turmas de forma dinâmica.
 - Botão para criar uma nova turma.
- 
- ![Alt Text](epic2-hist1.png)
+
+![Alt Text](epic2-hist1.png)
 
 **Backend**
-- Módulo gerenciador_turmas.py com funções para listar turmas.
- - Rotas de API:
- 	- /api/turmas/get: Retorna a lista de turmas disponíveis.
-	- /api/turmas/delete: Deletar uma turma.
 
+- Módulo gerenciador_turmas.py com funções para listar turmas.
+- Rotas de API:
+  - /api/turmas/get: Retorna a lista de turmas disponíveis.
+  - /api/turmas/delete: Deletar uma turma.
```

Comment:
Como essa ação de deleção vai ocorrer para uma determinada turma, ela precisa conter o ID de qual turma será deletada no path. Então, sugiro deixar isso explicito. 
api/turmas/delete/{id}
Apesar dessa será definido como nossa prática, é isso um problema em relação a ataque.  https://dev.to/bernardobrezende/devsecops-ataque-de-alteracao-de-parametros-da-requisicao-4ip9

Reply:
Ajustado no Backlog e no Markdown. Essa informação não deveria estar ai, visto que tem um Historia de usuario que trata de deletar uma turma.

Reply:
Legal, então pode tirar dai! vc tem razão.